### DISCUSSION TITLE
Mergify constantly stating that the configuration has changed

### DISCUSSION THREAD

**Discussion by manoelmarques** [Mergifyio/mergify#5054] [Q&A] [answered]
Hi,

After I changed the .mergify.yml to point to a new backport branch, automerge stopped  working and backport still tries to backport to the previous branch.
I noticed all new PRs have this Mergify message: "Configuration changed. This pull request must be merged manually"
However the .mergify.yml has already been changed and merged in a previous PR.
Any ideas of why this is happening ?
https://github.com/Qiskit/qiskit-nature/pull/939/checks

**Comment by jd** [comment]
Hi @manoelmarques 

That sounds like a bug, we're looking into it.

**Comment by manoelmarques** [comment, marked as answer]
It looks like is working again. 
Thanks a lot for your help.
